ci/container/build-ic.sh

```diff
@@ -113,7 +113,8 @@ rm -rf "$CANISTERS_DIR_FULL"
 rm -rf "$DISK_DIR_FULL"
 
 echo_green "Building selected IC artifacts"
-BAZEL_CMD="bazel build --config=local --ic_version='$VERSION' --ic_version_rc_only='$IC_VERSION_RC_ONLY'"
+BAZEL_CMD="bazel build --config=local --ic_version='$VERSION' --ic_version_rc_only='$IC_VERSION_RC_ONLY' "\
```

Comment:
shfmt is complaining about [this](https://dash.zh1-idx1.dfinity.network/invocation/21a87538-4092-41d0-91bf-2d64f4deb68b?target=%2F%2Fpre-commit%3Ashfmt-check&targetStatus=6). Maybe just update to have it in one line.

Reply:
Fixed (shfmt is happy and lines are not too long)